interface-definitions/load-balancing-inbound-ipvsadm.xml.in

```diff
@@ -0,0 +1,227 @@
+                      </valueHelp>
+                      <valueHelp>
+                        <format>wrr</format>
+                        <description>Weighted Round Robin</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>lc</format>
+                        <description>Least-Connection</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>wlc</format>
+                        <description>Weighted Least-Connection</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>lblc</format>
+                        <description>Locality-Based Least-Connection</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>lblcr</format>
+                        <description>Locality-Based Least-Connection with Replication</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>dh</format>
+                        <description>Destination Hashing</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>sh</format>
+                        <description>Source  Hashing</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>sed</format>
+                        <description>Shortest Expected Delay</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>nq</format>
+                        <description>Never Queue</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>fo</format>
+                        <description>Weighted Failover</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>ovf</format>
+                        <description>Weighted Overflow</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>mh</format>
+                        <description>Maglev Hashing</description>
+                      </valueHelp>
+                      <constraint>
+                        <regex>^(rr|wrr|lc|wlc|lblc|lblcr|dh|sh|sed|nq|fo|ovf|mh)$</regex>
+                      </constraint>
+                      <constraintErrorMessage>Load balancing algorithms must be rr, wrr, lc, wlc, lblc, lblcr, dh, sh, sed, nq, fo, ovf, or mh</constraintErrorMessage>
+                    </properties>
+                    <defaultValue>rr</defaultValue>
+                  </leafNode>
+                  <tagNode name="backend">
+                    <properties>
+                      <help>Back-end server</help>
+                      <valueHelp>
```

Comment:
Any reason the backend server IP address is not the tagNode value itself? We use this for RADIUS (inkl. priority) or other services, too.

Reply:
done